package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "fullstory-babel-plugin-annotate-react",
+  "name": "@fullstory/babel-plugin-annotate-react",
   "version": "0.1.0",
```

Comment:
What do you think about starting with v 1.0.0?

Reply:
I don't have a preference and will change it.